--- a/src/osg/ClipPlane.cpp
+++ b/src/osg/ClipPlane.cpp
@@ -46,6 +46,6 @@ void ClipPlane::apply(State&) const
 #if defined(OSG_GL_FIXED_FUNCTION_AVAILABLE) && !defined(OSG_GLES1_AVAILABLE)
     glClipPlane((GLenum)(GL_CLIP_PLANE0+_clipPlaneNum),_clipPlane.ptr());
 #else
-    OSG_NOTICE<<"Warning: ClipPlane::apply(State&) - not supported."<<std::endl;
+    //OSG_NOTICE<<"Warning: ClipPlane::apply(State&) - not supported."<<std::endl;
 #endif
 }
